resources/serverless/templates/service-account.yaml

```diff
@@ -1,12 +1,21 @@
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: {{ template "fullname" . }}-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tplValue" ( dict "value" .Values.global.commonLabels "context" . ) | nindent 4 }}
+    serverless.kyma-project.io/config: service-account
```

Comment:
Please remove this label :) This is only used for propagation below ServiceAccount :)